js/packages/web/babel.config.js

```diff
@@ -0,0 +1,19 @@
+module.exports = function (api) {
+  const isServer = api.caller((caller) => caller?.isServer)
+  const isCallerDevelopment = api.caller((caller) => caller?.isDev)
+ 
+   const presets = isCallerDevelopment
+    ? [
+        ['next/babel'],
+        [
+          '@babel/preset-react',
+          {
+            runtime: 'automatic',
+            importSource: !isServer ? '@welldone-software/why-did-you-render' : 'react',
+          },
+        ],
+      ]
+    : ['next/babel']
+ 
+   return { presets, plugins: ['macros'] }
+}
```

Comment:
```suggestion
}

```

Reply:
not super important i guess but a missing newline here

Reply:
I added newline here.